backend/tests/test_exporter.py

```diff
@@ -0,0 +1,75 @@
+import tempfile
+
+import openzim_uploader
+import pytest
+
+from backend.constants import BackendConf
+from backend.exporters import ExporterInterface
+from backend.exporters.kiwix_public_exporter import KiwixPublicExporter
+
+
+@pytest.mark.asyncio
+async def test_exporter_interface():
+
+    with pytest.raises(NotImplementedError):
+        for _ in await ExporterInterface().export():
+            ...
+
+
+@pytest.mark.asyncio
+async def test_incorrect_path(monkeypatch):
+    def mock_named_temporary_file(delete=False):
+        class MockNamedTemporaryFile(object):
+            def __init__(self, *args, **kwargs):
+                self.name = "/fakefile"
+
+        return MockNamedTemporaryFile()
+
+    with monkeypatch.context() as mp:
+        mp.setattr(tempfile, "NamedTemporaryFile", mock_named_temporary_file)
+        assert not await KiwixPublicExporter.export()
+
+
+@pytest.mark.asyncio
+async def test_incorrect_rsa_key(monkeypatch, ssh_private_key):
+    with monkeypatch.context() as mp:
+        mp.setattr(BackendConf, "private_key", ssh_private_key)
+    assert not await KiwixPublicExporter.export()
+
+
+@pytest.mark.asyncio
+async def test_incorrect_url(monkeypatch):
+    with monkeypatch.context() as mp:
+        mp.setattr(BackendConf, "upload_uri", "sftp://localhost:8080/data/test/")
```

Comment:
Your test is named `test_incorrect_url` yet the URL you are testing with in the test is identical to the one you are testing with in the next test which is a successful *upload*. Something is not right.